src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALL = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
**lottery is not a ball**